code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -6,70 +6,70 @@
 	w_class = WEIGHT_CLASS_SMALL
 	///Last person that touched this
 	var/mob/living/last_user
-	///how many charges do we have?
-	var/charge = 1
-	///Where we cannot create the rune?
-	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava))
 	///Is it in use?
 	var/in_use = FALSE
+	var/charge = 0
 
 /obj/item/forbidden_book/Destroy()
 	last_user = null
 	. = ..()
 
-
 /obj/item/forbidden_book/examine(mob/user)
 	. = ..()
 	if(!IS_HERETIC(user))
 		return
-	. += "The Tome holds [charge] charges."
-	. += "Use it on the floor to create a transmutation rune, used to perform rituals."
-	. += "Hit an influence in the black part with it to gain a charge."
-	. += "Hit a transmutation rune to destroy it."
+	//revise
+	//. += "The Tome holds [charge] charges."
+	//. += "Use it on the floor to create a transmutation rune, used to perform rituals."
+	//. += "Hit an influence in the black part with it to gain a charge."
+	//. += "Hit a transmutation rune to destroy it."
+	. += "Any mortal that reads this book will gain fascination. Baptise them with your Mansus Grasp to turn them into your disciples."
```

Comment:
this is a little confusing

Reply:
Ah just saw the code below. Should specify that you have to use the book on them in some way without being too OOC (Although if it is OOC just add <span class='notice'> and put it at the bottom like other things)